newsroom/navigations/navigations.py

```diff
@@ -0,0 +1,75 @@
+import newsroom
+import superdesk
+
+
+class NavigationsResource(newsroom.Resource):
+    """
+    Navigations schema
+    """
+    schema = {
```

Comment:
imo it's missing some field we could use for manual sorting

Reply:
Added `order` field